src/Pages/Home/HomePage.tsx

```diff
@@ -1,5 +1,131 @@
+import { useState } from 'react';
+import { LanguageSelector } from '../../components';
+const languages: string[] = [
```

Comment:
Do you think a constant variable inside of an individual const file would be better here?